add Admin questions (#193)
FidelusAleksander/ghcertified: content/questions/admin/question-093.md

---

> question: "How can you access, write, and share GitHub Packages within your projects?"

Reviewer: ```suggestion
question: "How can one authenticate to GitHub Packages?"
```

archetype: "questions"
title: "Question 093"
draft: false
---

> https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages#authenticating-to-github-packages

> 1. [x] By authenticating using a PAT with appropriate scopes, and configuring package client for use with GitHub Packages.
> 2. [ ] GitHub Packages can be accessed and shared publicly without any authentication.
> 1. [ ] Access and sharing are restricted to GitHub Enterprise accounts only.
> 1. [ ] By using the default GitHub login without needing any special authentication tokens.

Reviewer: ```suggestion
1. [x] By authenticating using a PAT with appropriate scope
1. [ ] GitHub Packages can be accessed and shared publicly without any authentication.
1. [ ] Access to GitHub Packages is restricted to GitHub Enterprise accounts only.
1. [ ] By using the default GitHub login without needing any special authentication tokens.
```
